lib/rules/rule-empty-line-before/index.js

```diff
@@ -52,7 +53,7 @@ const rule = function(expectation, options, context) {
       }
 
       // Ignore the first node
-      if (rule === root.first) {
+      if (isFirstNodeOfRoot(rule, root)) {
```

Comment:
Did you mean to pass `root` in as a second parameter?

Reply:
It was a mistake.